cmd/end2endtest/helpers.go

```diff
@@ -0,0 +1,402 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	vapi "go.vocdoni.io/dvote/api"
+	"go.vocdoni.io/dvote/apiclient"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+	"go.vocdoni.io/dvote/crypto/zk"
+	"go.vocdoni.io/dvote/log"
+	"go.vocdoni.io/dvote/types"
+	"go.vocdoni.io/dvote/util"
+	"go.vocdoni.io/proto/build/go/models"
+)
+
+const (
+	notWaitUntilNextBlock = "notWaitNextBlock"
+	waitUntilNextBlock    = "waitNextBlock"
+)
+
+func newTestElectionDescription() *vapi.ElectionDescription {
+	return &vapi.ElectionDescription{
+		Title:       map[string]string{"default": fmt.Sprintf("Test election %s", util.RandomHex(8))},
+		Description: map[string]string{"default": "Test election description"},
+		EndDate:     time.Now().Add(time.Minute * 20),
+
+		Questions: []vapi.Question{
+			{
+				Title:       map[string]string{"default": "Test question 1"},
+				Description: map[string]string{"default": "Test question 1 description"},
+				Choices: []vapi.ChoiceMetadata{
+					{
+						Title: map[string]string{"default": "Yes"},
+						Value: 0,
+					},
+					{
+						Title: map[string]string{"default": "No"},
+						Value: 1,
+					},
+				},
+			},
+		},
+	}
+}
+
+func (t e2eElection) createAccount(address string) (*vapi.Account, error) {
```

Comment:
i think i'm nitpicking on sugar here, but:
 all methods are `(t e2eElection)` except setupElection which is `(t *e2eElection)`
  i'd change all to be  `(t *e2eElection)`, consistent with the other E2E structs

Reply:
Yeah, only `setupElection` uses pointer because it assigns struct `e2eElection` values, but to be more consistent I'll update all methods to be `(t *e2eElection)`